From 004922e479b985e454fcb9f0b1da832e9b625fbd Mon Sep 17 00:00:00 2001
From: Salini Venate <salini.venate@intel.com>
Date: Wed, 28 Jun 2023 17:03:46 +0530
Subject: [PATCH] Commenting the sepolicy version

Commenting out the sepolicy version for now.This will
be reverted later.

Tracked-On: NA
Signed-off-by: Salini Venate <salini.venate@intel.com>
---
 groups/device-specific/celadon_ivi/manifest.xml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/groups/device-specific/celadon_ivi/manifest.xml b/groups/device-specific/celadon_ivi/manifest.xml
index ccc783f..e834dd4 100644
--- a/groups/device-specific/celadon_ivi/manifest.xml
+++ b/groups/device-specific/celadon_ivi/manifest.xml
@@ -143,7 +143,7 @@
         </interface>
     </hal>
     <!--<kernel target-level="5"/>-->
-    <sepolicy>
+    <!--<sepolicy>
         <version>33.0</version>
-    </sepolicy>
+    </sepolicy>-->
 </manifest>
-- 
2.17.1

